pingdom-checks/veteran-verification.ping

```diff
@@ -25,3 +26,29 @@ pingdom save-check \
   -a group=veteran-verification \
   -a userids_csv="$STATUSPAGE_SANDBOX_VETERAN_VERIFICATION" \
   -a integrationids_csv="$VETERAN_VERIFICATION_SLACK_ID,$PAGER_DUTY"
+
+#sandbox
+pingdom save-check \
+  --template https-public-200 \
+  -a name=sandbox-veteran-verification-health \
```

Comment:
This needs to be renamed to sandbox-veteran-verification-v2-health, otherwise it will conflict with the existing healthchecks.